packages/select/src/components/select/suggest.tsx

```diff
@@ -111,33 +103,32 @@ export class Suggest<T> extends React.PureComponent<ISuggestProps<T>, ISuggestSt
     }
 
     private renderQueryList = (listProps: IQueryListRendererProps<T>) => {
-        const { inputValueRenderer, inputProps = {}, popoverProps = {} } = this.props;
-        const { isTyping, selectedItem, query } = this.state;
+        const { inputProps = {}, popoverProps = {} } = this.props;
+        const { isOpen, selectedItem } = this.state;
         const { handleKeyDown, handleKeyUp } = listProps;
-        const inputValue: string = isTyping ? query : selectedItem ? inputValueRenderer(selectedItem) : "";
 
+        const selectedItemText = selectedItem ? this.props.inputValueRenderer(selectedItem) : "";
         return (
             <Popover
                 autoFocus={false}
                 enforceFocus={false}
-                isOpen={this.state.isOpen}
+                isOpen={isOpen}
                 position={Position.BOTTOM_LEFT}
                 {...popoverProps}
                 className={classNames(listProps.className, popoverProps.className)}
                 onInteraction={this.handlePopoverInteraction}
                 popoverClassName={classNames(Classes.SELECT_POPOVER, popoverProps.popoverClassName)}
                 onOpened={this.handlePopoverOpened}
-                onClosing={this.handlePopoverClosing}
             >
                 <InputGroup
-                    placeholder="Search..."
-                    value={inputValue}
+                    placeholder={isOpen && selectedItemText ? selectedItemText : "Search..."}
```

Comment:
`""`, which you set as the default on line 110 is falsy. is that what you intend?

Reply:
yes it should say "Search..." instead of ""